components/Collections/src/MDCCollectionViewEditing.h

```diff
@@ -55,4 +55,15 @@
  */
 - (void)setEditing:(BOOL)editing animated:(BOOL)animated;
 
+/**
+ Updates the position of the cell being reordered to match the user's touch location. This method
+ is a no-op if a cell is not currently being reordered.
+ 
+ While the reordering cell's location is updated automatically to match the user's touch location,
+ this method allows for the explicit update of the cell's position which may be required when the
+ cell's location changes due to a layout update (such as performing collection view changes while
+ the user is reordering).
+ */
+- (void)updateReorderCellPosition;
```

Comment:
These docs describe when to use this API — can we also provide additional context such as which class or method we might call this API from?

Reply:
Would something like this be what you're asking for:

### Example

- (void)collectionView:(nonnull UICollectionView *)collectionView
    willBeginDraggingItemAtIndexPath:(nonnull NSIndexPath *)indexPath {
  [self.collectionView performBatchUpdates: ^{
    // Perform set up for moving, such as collapsing sections or any other animations pre-move.
  } completion:^(BOOL finished) {
    // Ensure that the reordering cell is under user's touch after the updates are performed.
    [self.editor updateReorderCellPosition];
  }];
}

Or would it be sufficient to add:

"For example, you can call this in the the MDCCollectionViewEditingDelegate's implementation of -collectionView:willBeginDraggingItemAtIndexPath:, after performing updates to the collection view in preparation for the user's move of an item."

Reply:
> "For example, you can call this in the the MDCCollectionViewEditingDelegate's implementation of -collectionView:willBeginDraggingItemAtIndexPath:, after performing updates to the collection view in preparation for the user's move of an item."

This is super helpful!